packages/e2e/cypress/e2e/run/create-pipelinerun.cy.js

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2022 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const pipeline = `
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: simple-pipeline
+  namespace: e2e
+spec:
+  tasks:
+    - name: hello
+      taskSpec:
+        steps:
+          - name: echo
+            image: busybox
+            script: |
+              #!/bin/ash
+              echo "Hello World!"
+`;
+describe('Create Simple Pipeline Run', () => {
+  before(() => {
+    cy.exec('kubectl version --client');
+    cy.exec('kubectl create namespace e2e || true');
+    cy.exec(`echo "${pipeline}" | kubectl apply -f -`);
+  });
+  it('should create pipelinerun', function () {
+    cy.visit(
+      `/#/pipelineruns/create?namespace=e2e&pipelineName=simple-pipeline`
+    );
+    cy.contains('button', 'Create').click();
+    cy.contains('simple-pipeline').first().click();
+
+    cy.get('header[class="tkn--pipeline-run-header"]', { timeout: 10000 })
```

Comment:
I'm curious where the timeout values here came from, was it from experimentation?
We'll need to keep an eye on these to ensure we don't end up with a flaky test.

Reply:
It was experimentation over many runs (but I have done that for pipeline with workspaces)

reworked
* remove timeout for header (let's assume I my system has been hanging)
* decrease timeout for status from 15s to 10s
* add retry = 3
https://cypress.io/blog/2020/12/03/retry-rerun-repeat/

Reply:
Sorry I should have been clearer, my comment was just pointing out that we'll need to be aware of this and account for CI potentially being slower than a local cluster, not requesting a change in strategy.

I would prefer to increase the timeout for the status and remove the retries. Especially in CI where we create a new cluster and registry it's likely to take longer to run the first time due to pulling the image used in the pipeline. Subsequent attempts will most likely be faster as the image has already been pulled which is why the retries succeed. Using retries to accommodate this also has the side effect of creating multiple runs which we don't really need.

We should also consider cleanup after a successful run and delete it but that can be done later.

Reply:
Please also squash your commits when done so this is ready to merge.

Reply:
Ok,
* timeout increased to `15s` for status wait
   * on my pc and CI it takes 6s for everything (but image is pulled before that because `image: busybox` is used in static resources and those tests)
![image](https://user-images.githubusercontent.com/9289172/200097868-557a6da9-630c-4fb7-a138-c58e82bb158e.png)
* retries removed

@AlanGreene, note, I have changed permission for `~/.kube/config` file to `644` otherwise permission denied on CI. I don't like that much for local runs, because it change permissions for common user kubeconfig file
Other options:
* copy `~/.kube/config` file to `~/.kube/tekton`, export KUBECONFIG=~/.kube/tekton`, change permissions and cleanup somewhere (don't know where for CI)
* `export KUBECONFIG=~/.kube/tekton`, then `kind export kubeconfig --name xxxx`, then `chmod 644 $KUBECONFIG` and finally mount `-v $KUBECONFIG:/home/node/.kube/config` (but it is not clear for me where cluster is created for CI). But it may have other problems for user, e.g. it will require to specify kubeconfig for each terminal session. For example, I use this kubecluster for development purposes and I don't tear down it after runs.